include/grpc/grpc_security.h

```diff
@@ -834,30 +838,73 @@ typedef void (*grpc_tls_on_credential_reload_done_cb)(
     grpc_tls_credential_reload_arg* arg);
 
 /** A struct containing all information necessary to schedule/cancel a
-    credential reload request.
-    - cb and cb_user_data represent a gRPC-provided
-      callback and an argument passed to it.
-    - key_materials_config is an in/output parameter containing currently
-      used/newly reloaded credentials. If credential reload does not result in
-      a new credential, key_materials_config should not be modified. The same
-      key_materials_config object can be updated if new key materials is
-      available.
-    - status and error_details are used to hold information about
-      errors occurred when a credential reload request is scheduled/cancelled.
-    - config is a pointer to the unique grpc_tls_credential_reload_config
-      instance that this argument corresponds to.
-    - context is a pointer to a wrapped language implementation of this
-      grpc_tls_credential_reload_arg instance.
-    - destroy_context is a pointer to a caller-provided method that cleans
-      up any data associated with the context pointer.
+    credential reload request. To help understand each fields, we mark each
+    field with labels. Here are their explanations
+    -------------------------label explanation----------------------------------
+    [USER-MANAGED]: fields that are managed directly by the end users
+    [SYSTEM-PROVIDED]: fields that needed to retrieve implementations in C core
```

Comment:
What implementations do you mean?

Reply:
This may need to be reworded to clarify? e.g. "fields whose values are provided by the gRPC stack and used (consumed) by the C core implementation" assuming that is what is meant?

Reply:
Yes that's exactly what I meant. Thanks for the suggestion!